src/MizarBase/include/MizarBase/Titles.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_BASE_TITLES_H_
+#define MIZAR_BASE_TITLES_H_
+
+#include <QString>
+
+#include "MizarBase/BaselineOrComparison.h"
+
+namespace orbit_mizar_base {
+
+static const orbit_mizar_base::Baseline<QString> kBaselineTitle =
+    orbit_mizar_base::Baseline<QString>(QStringLiteral("Baseline"));
+static const orbit_mizar_base::Comparison<QString> kComparisonTitle =
+    orbit_mizar_base::Comparison<QString>(QStringLiteral("Comparison"));
```

Comment:
This is fine for your current usage, but depending on where you want to use the constants in the future you could quickly end up in static initialization order fiasco (I think - I'm not entirely sure whether defining those in a header could save you from that. If you don't know for sure I wouldn't count on it.).

I'd recommend to use some type that can be constant initialized - like `std::string_view` and then create QStrings on-the-fly when you need them. This might have more overhead due to QStrings being constructed multiple times, but I assume you're not planning on constructing such QStrings in a hot loop.

Reply:
TIL static initialization order fiasco. Thanks. 

Let's replace the two constants with two free functions returning a ref to a local static object.